Bottom: 6e1d365e7e919adda7bf14bf91a0d7670e36945f
Top:    77049c342d7ee1651010da7b67838f9db66921e5
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-29 14:13:16 -0500

Refresh of TESTING--VirtualMachine

---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index 280d0724..2cf62457 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -52,10 +52,6 @@
     role: sshd
     tags:
     - sshd
-  - name: Run bacula-restores symlinks
-    role: bacula-restores
-    tags:
-    - bacula_restores
   - name: Run common Docker tasks
     role: docker-common
     tags:
@@ -68,6 +64,10 @@
   become_user: root
   become_method: sudo
   roles:
+  - name: Run bacula-restores symlinks
+    role: bacula-restores
+    tags:
+    - bacula_restores
   - name: Configure Docker networking
     role: docker-networking
     network_connection: '{{ daedalus_net_connection }}'
@@ -102,7 +102,7 @@
     tags:
     - exim4_container
   - name: Build and start the GitLab Docker container
-    role: docker-gitlab
+    role: gitlab-omnibus
     gitlab_db_user: novatech
     tags:
     - gitlab_container
